### PR TITLE
Trivial: Fix doc

### PR DESCRIPTION
**Pull request by kchulj**

### CODE REVIEW

File: src/modules/storage/LedgerStorage.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*******************************************************************************
 
-    The class that create and insert and read the ledger into the database.
+    The class that creates, inserts, and reads the ledger into the database.
```
Comment (hewison-chris) [inline src/modules/storage/LedgerStorage.ts]: You have an extra comma.

**Review by hewison-chris — CHANGES_REQUESTED** [review]
Small change otherwise good.

File: src/modules/storage/LedgerStorage.ts (hunk 1 above)
Comment (kchulj) [reply src/modules/storage/LedgerStorage.ts]: Fixed :)

**Review by hewison-chris — APPROVED** [review]
LGTM

**Review by TrustHenry — APPROVED** [review]

**Pull request merged**